en/docs/design/api-policies/gateway-policies/creating-and-uploading-using-integration-studio.md

```diff
@@ -1,6 +1,6 @@
 # Creating and Uploading Custom Mediation Policies using WSO2 Integration Studio
 
-You can design all custom mediation policies using a tool such as WSO2 Integration Studio and then store the policy in the registry which can be later deployed to the Gateway.
+You can design all custom mediation policies using a tool such as WSO2 Integration Studio and then store the policy in the registry, which can be later deployed to the Gateway.
```

Comment:
We have to remove this. This flow does not work with the new feature

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/5528/commits/003de52a4413e93e7f9cec9b530e8793e80fdf33